java/src/jni/h5jni.h

```diff
@@ -257,6 +257,23 @@
     do {                                                                                                     \
         (*envptr)->ReleaseStringUTFChars(envptr, pinnedString, stringToRelease);                             \
     } while (0)
+/*
+ * Above String macros may be incorrect, suggested code for getting a cstr from java
+ * int jstr_to_cstr(JNIEnv *jenv, jstring j_str, char *c_str, size_t cstr_len)
```

Comment:
We should decide what to do about this instead of just adding a commented out section of code